src/views/creating_accommodation_facilities/CreateAccommodation.jsx

```diff
@@ -0,0 +1,865 @@
+	constructor(props) {
+		super(props);
+		this.state = {
+			accommodationName: '',
+			description: '',
+			locationId: '',
+			owner: '',
+			category: '',
+			images:  this.props.images,
+			amenities: [],
+			services: [],
+			serviceName: '',
+			amenityName: '',
+			disabled: true,
+			AmenitiesDisabled: true,
+			SaveDisabled: true,
+			SnackbarMessage: '',
+			files: [],
+			RoomFiles: [],
+			open: false,
+			roomChildCard: 2,
+			roomImagesState: [],
+			roomImages: [],
+			rooms: [
+				{
+					numberOfRoom: '',
+					typeId: '',
+					price: '',
+					currency: '',
+					roomTypeImageUrl: [],
+				},
+				{
+					numberOfRoom: '',
+					typeId: '',
+					price: '',
+					currency: '',
+					roomTypeImageUrl: [],
+				},
+				{
+					numberOfRoom: '',
+					typeId: '',
+					price: '',
+					currency: '',
+					roomTypeImageUrl: [],
+				},
+			],
+			activeItemIndex: 0,
+		deleted:false
+		};
+	}
+
+	handleClick(event) {
+		const {
+			accommodationName,
+			description,
+			locationId,
+			owner,
+			category,
+			images,
+			rooms,
+			amenities,
+			services,
+		} = this.state;
+		const data = {
+			accommodationName: accommodationName,
+			description: description,
+			locationId: locationId,
+			owner: owner,
+			category: category,
+			images: this.props.images,
+			rooms: rooms,
+			services: services,
+			amenities: amenities,
+		};
+		const { createAccomodationFacility } = this.props;
+		createAccomodationFacility(data);
+	}
+
+	componentDidUpdate(prevProps) {
+		if (this.props.data !== prevProps.data) {
+			history.push('/accommodations');
+		}
+	}
+	uploadAccommodationImage = async (e )=> {
+		this.state.files.push(...e.target.files);
+		const files = [...e.target.files];
+		files.map((file, index) => {
+			const data = new FormData();
+			data.append('file', files[index]);
+			data.append('upload_preset', 'barefootnomad');
+
+		this.props.uploadNewImageOnCloud(data,index);
+		
+		});
+		this.setState({
+			files: [],
+		});
+	};
+	HandleUploadRoomImage = async (e, cardId) => {
+		this.state.files.push(...e.target.files);
+		const files = [...e.target.files];
+		files.map((file, index) => {
+			const data = new FormData();
+			data.append('file', files[index]);
+			data.append('upload_preset', 'barefootnomad');
+			this.props.uploadNewRoomImageOnCloud(data, cardId);
+		});
+	};
+	handleChange(event) {
+		this.setState({
+			[event.target.name]: event.target.value,
+			SaveDisabled: false,
+		});
+
+		if (
+			this.state.accommodationName !== '' &&
+			this.state.locationId !== '' &&
+			this.state.owner !== ''
+		) {
+			this.setState({
+				SaveDisabled: false,
+			});
+		}
+	}
+	handleChangeRooms = (event, value, Index1, name) => {
+		const rooms = this.state.rooms;
+		rooms.map((room, index) => {
+			if (index === Index1) {
+				if (event.target.id !== 'standard-search') {
+					room[name] = value;
+					// if (name === 'roomTypeImageUrl') {
+					// 	room[name].push(value);
+					// }
+				}
```

Comment:
@mujohn26 kindly remove the commented lines

Reply:
@jo-rdan Thanks for the feedback I changed.